docs/configuration.rst

```diff
@@ -28,10 +28,49 @@ Top level keys
 
 Custom fragment types
 ---------------------
+``Towncrier`` allows defining custom fragment types. Custom fragment types
+will be used instead ``towncrier`` default ones, they are not combined.
 
-Custom fragment types can be including in the
-pyproject.toml.
-Each custom type (``[[tool.towncrier.type]]``) has the following keys:
+
+Custom fragment types can be including in the pyproject.toml in two different ways:
+
+Defining custom fragment types with toml tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Users can create their own custom fragment types by adding tables to
+the pyproject.toml named ``[tool.towncrier.type.<a custom fragment type>]``.
+
+These tables may include the following optional keys:
+
+ * ``name``: The description of the fragment type, as it must be included
+   in the news file. If omitted, it defaults to  its  fragment type,
+   but capitalized.
+ * ``showcontent``: Whether if the fragment contents should be included in the
+   news file. If omitted, it defaults to ``true``
+
+
+For example:
+
+.. code-block:: toml
+
+
+        [tool.towncrier]
+        [tool.towncrier.type.chores]
+        name = "Other Tasks"
+        showcontent = False
+
+        [tool.towncrier.type.deprecations]
+        # This fragment type will be named "Deprecations"
+        # and it will show it contents.
+
+
+Defining custom fragment types with an array of toml tables
```

Comment:
@adiroiban  do you want to indicate this section of the docs as deprecated?

Reply:
Yes... or not document it at all :) or only add a very brief info with the mention that this is no longer recommend.